src/components/OfflineWithFeedback.tsx

```diff
@@ -0,0 +1,139 @@
+ * care of adding the appropriate styles for pending actions and displaying the dismissible error.
+ */
+
+type OfflineWithFeedbackProps = ChildrenProps & {
+    /** The type of action that's pending  */
+    pendingAction: OnyxCommon.PendingAction | null;
+
+    /** Determine whether to hide the component's children if deletion is pending */
+    shouldHideOnDelete?: boolean;
+
+    /** The errors to display  */
+    errors?: OnyxCommon.Errors | null;
+
+    /** Whether we should show the error messages */
+    shouldShowErrorMessages?: boolean;
+
+    /** Whether we should disable opacity */
+    shouldDisableOpacity?: boolean;
+
+    /** A function to run when the X button next to the error is clicked */
+    onClose?: () => void;
+
+    /** Additional styles to add after local styles. Applied to the parent container */
+    style?: StyleProp<ViewStyle>;
+
+    /** Additional styles to add after local styles. Applied to the children wrapper container */
+    contentContainerStyle?: StyleProp<ViewStyle>;
+
+    /** Additional style object for the error row */
+    errorRowStyles?: StyleProp<ViewStyle>;
+
+    /** Whether applying strikethrough to the children should be disabled */
+    shouldDisableStrikeThrough?: boolean;
+
+    /** Whether to apply needsOffscreenAlphaCompositing prop to the children */
+    needsOffscreenAlphaCompositing?: boolean;
+
+    /** Whether we can dismiss the error message */
+    canDismissError?: boolean;
+};
+
+type StrikethroughProps = Partial<ChildrenProps> & {style: Array<ViewStyle | TextStyle | ImageStyle>};
+
+/**
+ * This method applies the strikethrough to all the children passed recursively
+ */
+function applyStrikeThrough(children: React.ReactNode): React.ReactNode {
+    return React.Children.map(children, (child) => {
+        if (!React.isValidElement(child)) {
+            return child;
+        }
+
+        const props: StrikethroughProps = {
+            style: StyleUtils.combineStyles(child.props.style, styles.offlineFeedback.deleted, styles.userSelectNone),
+        };
+
+        if (child.props.children) {
+            props.children = applyStrikeThrough(child.props.children);
+        }
+
+        return React.cloneElement(child, props);
+    });
+}
+
+function OfflineWithFeedback({
+    pendingAction = null,
+    canDismissError = true,
+    contentContainerStyle,
+    errorRowStyles,
+    errors = null,
+    needsOffscreenAlphaCompositing = false,
+    onClose = () => {},
+    shouldDisableOpacity = false,
+    shouldDisableStrikeThrough = false,
+    shouldHideOnDelete = true,
+    shouldShowErrorMessages = true,
+    style,
+    ...props
+}: OfflineWithFeedbackProps) {
+    const {isOffline} = useNetwork();
+
+    const hasErrors = !isEmptyObject(errors);
+    // Some errors have a null message. This is used to apply opacity only and to avoid showing redundant messages.
+    const errorMessages = lodashOmitBy(errors, (e) => e === null);
```

Comment:
How about implementation without lodash?

Reply:
FYI Original implementation used omit, not omitBy:
```ts
    const errorMessages = _.omit(props.errors, (e) => e === null);
````

Reply:
@blazejkustra lodash omit is used only for omits by keys 
<img width="1263" alt="image" src="https://github.com/Expensify/App/assets/25230417/bb75a682-7c25-4af6-8d88-50d14db5cc32">
for omit by some function `omitBy` need to be used 
<img width="1267" alt="image" src="https://github.com/Expensify/App/assets/25230417/6e4ad2f1-2f95-4235-bec2-0508cf686970">